packages/commands/turbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": ["dist/**"],
+      "dependsOn": ["^build"],
+      "outputMode": "new-only"
+    },
+    "test": {
+      "cache": false,
+      "outputMode": "new-only",
+      "dependsOn": ["build"]
+    },
```

Comment:
You shouldn't need these because they're already defined in the root `turbo.json`!

Reply:
I _slightly_ modified them:
Root `build` task marks `build/**` as an output and this one doesn't write to there, so I figured best to not include it as a cache output.
Root `test` depends on `^build` , which I understand to mean just waiting on dependencies to build. But we need the actual `build` task for this package to run. Maybe that's a bug in the root config?

Reply:
Oh and also, this `test` task doesn't write to `coverage`, so I wanted to remove that.